server/src/main/java/com/paypal/selion/pojos/BrowserStatisticsCollection.java

```diff
@@ -0,0 +1,132 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2016 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.pojos;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+
+import com.google.gson.Gson;
+import com.paypal.selion.logging.SeLionGridLogger;
+
+/**
+ * <code>BrowserStatisticsCollection</code> stores the all the supported browsers by the Grid and Nodes collectively. It
+ * stores the maximum available instances of a particular browser in the Grid - Nodes farm along with the waiting
+ * requests for a particular browser. This class is not thread safe, the clients are expected to create individual
+ * instances of this class in separate threads.
+ * 
+ */
+public class BrowserStatisticsCollection {
+
+    private static final SeLionGridLogger logger = SeLionGridLogger.getLogger(BrowserStatisticsCollection.class);
+
+    private List<BrowserStatistics> browserStatisticsList;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Private field 'browserStatisticsList' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2103178712/issues/source?bid=2928425&fileBranchId=3143459#l37)